src/main/java/Numbers.java

```diff
@@ -0,0 +1,23 @@
+public class Numbers {
+
+    public boolean checkIfNumberCanBeDividedByTwo(int givenNumber) {
+        return givenNumber % 2 == 0;
+
+    }
+
+    public int getSumOfDigits(int givenNumber) {
+        int numberOfDigits = String.valueOf(givenNumber).length();
+        int sumOfDigits = 0;
+        int[] arrayOfDigits = new int[numberOfDigits];
+
+        for (int i = 0; i <= numberOfDigits - 1; i++){
+            arrayOfDigits[i] = givenNumber / (int)Math.pow(10, numberOfDigits - 1 - i);
+            givenNumber = givenNumber - (int)Math.pow(10, numberOfDigits - 1 - i) * arrayOfDigits[i];
+            sumOfDigits = sumOfDigits + arrayOfDigits[i];
```

Comment:
O kurcze, ale przekombinowane:D

1 pytanie - czemu osobne repo?

Reply:
Co do rozwiązania, podpowiedź: modulo zwraca resztę z dzielenia. 345 % 10 = 5.

Spróuj trochę to uprościć. ;)

Reply:
Osobne repo żeby zachować porządek, strasznie dużo zrobiło się tych wszystkich plików i jak chce wrócić do jakich zadań z przeszłości to trudno było mi je znaleźć. Czy może lepszym rozwiązaniem było by umieszczanie zadań z każdej lekcji w osobnym package?

Kod uproszczony :)